src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,18 @@
-export const UserInfo = () => {};
+// import './UserInfo.scss';
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
+);
```

Comment:
The component uses optional chaining to handle the possibility of `user` being `null`. However, consider providing a fallback UI or message in case `user` is `null`, to improve user experience and avoid rendering an empty link.